tests/ops/op_math/test_controlled_decompositions.py

```diff
@@ -87,26 +87,21 @@ def test_invalid_op_error(self):
         ):
             _ = ctrl_decomp_zyz(qml.CNOT([0, 1]), [2])
 
-    def test_invalid_num_controls(self):
-        """Tests that an error is raised when an invalid number of control wires is passed"""
-        with pytest.raises(
-            ValueError,
-            match="The control_wires should be a single wire, instead got: 2",
-        ):
-            _ = ctrl_decomp_zyz(qml.X([1]), [0, 1])
-
     su2_ops = [
         qml.RX(0.123, wires=0),
         qml.RY(0.123, wires=0),
         qml.RZ(0.123, wires=0),
         qml.Rot(0.123, 0.456, 0.789, wires=0),
     ]
 
-    unitary_ops = [
+    special_unitary_ops = [
         qml.Hadamard(0),
         qml.PauliZ(0),
```

Comment:
Also, det([[1,0],[0,-1]]) == -1.... So is it actually special unitary or am I missing something?

Reply:
ahh, my bad! forgot to fix this before pushing :(